apiary.apib

```diff
@@ -6,6 +6,31 @@ OpenGrok RESTful API documentation. The following endpoints are accessible under
 
 Besides `/suggester` and `/search` endpoints, everything is accessible from `localhost` only.
 
+## Annotation [/annotation{?path}]
+
+### Get annotation for a file [GET]
+
++ Parameters
+  + path (string) - path of file, relative to source root
+
++ Response 200 (application/json)
+  + Body
+
+            [
+              {
+                "number": 1,
+                "revision": "e2736ff6",
+                "author": "Vladimir Kotal",
+                "desc": "changeset:&nbsp;e2736ff6&lt;br/&gt;summary:&nbsp;add&nbsp;microbadges&lt;br/&gt;user:&nbsp;Vladimir&nbsp;Kotal&nbsp;&#60;vlada@devnull.cz&#62;&lt;br/&gt;date:&nbsp;Fri&nbsp;Jun&nbsp;28&nbsp;21:33:41&nbsp;CEST&nbsp;2019"
```

Comment:
can have long naming "description"

Reply:
I wouldn't expect a html code inside json in a description field, is it easy to get rid of it?

Reply:
it turns out it is.